app/User.php

```diff
@@ -30,4 +30,13 @@ class User extends Authenticatable
 
     protected $test = 'testtest';
     protected $test2 = 'test2test2';
+
+    private $check_WIP = false;
+
+    private function newPR() {
+        $test = 'not-good';
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Rename "$test" which has the same name as the field declared at line 31. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://relisonar.relipasoft.com:9000/coding_rules#rule_key=php%3AS1117)
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused "$test" local variable. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://relisonar.relipasoft.com:9000/coding_rules#rule_key=php%3AS1481)